docs/tasks/debug-application-cluster/audit.md

```diff
@@ -25,36 +27,111 @@ answer the following questions:
     - from where was it initiated?
     - to where was it going?
 
-NOTE: Currently, Kubernetes provides only basic audit capabilities, there is still a lot
-of work going on to provide fully featured auditing capabilities (see [this issue](https://github.com/kubernetes/features/issues/22)).
+Audit is composed of two phases. Policy, which determines what's recorded, and backends,
+which persist records. Backend implementations include logs files and webhooks.
+
+As of 1.7 audit features require the `AdvancedAuditing` feature gate on the
+[kube-apiserver][kube-apiserver], which can be enabled through the `--feature-gates`
+flag.
 
-Kubernetes audit is part of [kube-apiserver](/docs/admin/kube-apiserver) logging all requests
-coming to the server. Each audit log contains two entries:
+```
+--feature-gates=AdvancedAuditing=true
+```
 
-1. The request line containing:
-    - unique id allowing to match the response line (see 2)
-    - source ip of the request
-    - HTTP method being invoked
-    - original user invoking the operation
-    - original user's groups info
-    - impersonated user for the operation
-    - impersonated groups info
-    - namespace of the request or <none>
-    - URI as requested
-2. The response line containing:
-    - the unique id from 1
-    - response code
+If disabled, the legacy 1.6 behavior is preserved.
 
-Example output for user `admin` asking for a list of pods:
```

Comment:
Left the current documentation relatively unchanged and called out the `AdvancedAuditing` feature flag more prominently.